### DISCUSSION TITLE
Orin NX build is falling under secondary partition.

### DISCUSSION THREAD

**Discussion by isaacporras** [OE4T/meta-tegra#1482] [General]
Hi, I'm currently working with an Orin NX 8GB custom carrier board.
I'm currently seing that it's booting into the secondary partition (UEFI INTEL SSDPEKNW020T8 PHNH037200CN2P0C 1 2), which prevents me to apply device tree overlays. Is there a way that the board boots directly after flash to the primary partition (UEFI INTEL SSDPEKNW020T8 PHNH037200CN2P0C 1) without requiring me to getting into the Boot Manager GUI menu?

![OrinNX_Boot manager](https://github.com/OE4T/meta-tegra/assets/31218031/f399ab91-5b22-4ae7-bbd7-615dd7fe6af1)

Thanks

**Comment by madisongh** [comment]
Can you tell us a bit more about your setup...

Which branch are you using?

Are you using a customized flash layout, or the stock L4T layout?  If customized, what did you change?  What is on partition 2?

How are you trying to apply the DTB overlays?

I vaguely recall there being some UEFI issues around boot ordering/enumeration in earlier R35 L4T releases, but I think they've all been fixed in the R35.4.1 release (except for RCM booting, which we add a patch for).  And if I boot my Orin NX off an NVMe drive, I only see one entry in the boot manager for it.
